tests/unit/services/cookies-test.js

```diff
@@ -396,6 +396,20 @@ describe('CookiesService', function() {
         expect(this.subject().read('test1')).to.be.empty;
         expect(this.subject().read('test2')).to.be.empty;
       });
+
+      it('clears all cookies with a specified path', function() {
+        let path = window.location.pathname;
+        document.cookie = `test1=${randomString()};path=${path}`;
+        document.cookie = `test2=${randomString()};`;
+
+        expect(this.subject().read('test1')).not.to.be.empty;
+        expect(this.subject().read('test2')).not.to.be.empty;
+
+        clearAllCookies({ path });
+
+        expect(this.subject().read('test1')).to.be.empty;
+        expect(this.subject().read('test2')).not.to.be.empty;
+      });
```

Comment:
It'd be great to add tests for `domain` and `secure` here as well 🙏

Reply:
This doesn't actually make sense as we cannot see and thus clear cookies for other domains anyway or cookies with `secure` if we're currently not on `https:` 🤦‍♂️